tests/sentry/incidents/test_charts.py

```diff
@@ -128,30 +128,44 @@ def test_eap_alert(self, mock_client_get: MagicMock, mock_generate_chart: MagicM
 
 class FetchOpenPeriodsTest(TestCase):
     @freeze_time(frozen_time)
-    @patch("sentry.incidents.charts.client.get")
+    @with_feature("organizations:incidents")
```

Comment:
Without this we were actually hitting a 404 hitting the incidents endpoint, but the test didn't care to check